Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -133,9 +137,11 @@ void CheckSendRate()
                 m_SendTimer = Time.time + syncInterval;
 
                 NetworkWriter writer = new NetworkWriter();
-                WriteParameters(writer, true);
-
-                SendAnimationParametersMessage(writer.ToArray());
+                if (WriteParameters(writer, true))
+                {
+                    SendAnimationParametersMessage(writer.ToArray());
+                    Debug.Log("Thing happened");
```

Comment:
please remove the debug.log

Reply:
Oh wow, how could I have missed that. Thanks for spotting that.